fdbserver/PaxosConfigConsumer.actor.cpp

```diff
@@ -297,21 +328,48 @@ class PaxosConfigConsumerImpl {
 						}
 					}
 					self->lastSeenVersion = committedVersion;
-					broadcaster->applyChanges(reply.changes, committedVersion, reply.annotations);
+					broadcaster->applyChanges(reply.changes,
+					                          committedVersion,
+					                          reply.annotations,
+					                          self->getCommittedVersionQuorum.getReadReplicas());
 					// TODO: Catch error_code_process_behind and retry with
 					// the next ConfigNode in the quorum.
+				} else if (committedVersion == self->lastSeenVersion) {
+					broadcaster->applyChanges({}, -1, {}, self->getCommittedVersionQuorum.getReadReplicas());
```

Comment:
This addition allows the broadcaster to update its known "alive" replicas even when the replicas respond to change requests with no new changes. It is maybe a little bit hacky. Could potentially do this better.

Reply:
Moved the common update code to its own function.